src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Subscriptions/DependenciesRuleHandlerBase.cs

```diff
@@ -71,16 +59,16 @@ public virtual bool ReceiveUpdatesWithEmptyProjectChange
         {
             if (projectChanges.TryGetValue(UnresolvedRuleName, out IProjectChangeDescription unresolvedChanges))
             {
-                HandleChangesForRule(false /*unresolved*/,
-                    unresolvedChanges, context, ruleChangeContext,
-                        (itemSpec) => { return true; });
+                HandleChangesForRule(
+                    resolved: false, unresolvedChanges, context, ruleChangeContext,
+                    itemSpec => true);
```

Comment:
Not introduce, but a name for what this delegate is would be nice too.

Reply:
Done.